fhir-server/liberty-config/server.xml

```diff
@@ -93,10 +93,12 @@
         <fileset dir="${shared.resource.dir}/lib/fhir" includes="*.jar"/>
     </library>
 
+    <!-- Since 4.5.0: Enable the fhirProxyDataSource only if you need the legacy datasource behavior
     <dataSource id="fhirProxyDataSource" jndiName="jdbc/fhirProxyDataSource" type="javax.sql.XADataSource" validationTimeout="30s">
         <jdbcDriver javax.sql.XADataSource="com.ibm.fhir.persistence.proxy.FHIRProxyXADataSource" libraryRef="fhirSharedLib"/>
         <connectionManager/>
     </dataSource>
+    -->
```

Comment:
Are you pushing this into a default? it seems like this should remain for one version or be moved completely to a configDropin default.

Reply:
Right, currently all the SITs are using the new mechanism, but I'm planning to revert the Db2 test to use the old mechanism to maintain coverage, and this will be uncommented to support that.

Reply:
Db2 configuration has been reverted to ensure we have continued coverage of the old-style configuration in SITs.